### PR TITLE
e2e: creates a separate package for selectors

### PR DESCRIPTION
**Pull request by hugohaggmark**
**What this PR does / why we need it**:
To remove circular dependencies between e2e => ui => e2e and such, this PR attempts to solve this.

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

**Comment by hugohaggmark** [comment]
Fails right now on bottom pane nav being broken.

**Comment by hugohaggmark** [comment]
Not super happy to have same `Pages` & `Components` structure in two places so any suggestions would be great!

**Review by dprokop — COMMENTED** [review]
Great! I think @stevenvachon will be happy ;)

**Comment by dprokop** [comment]
> Not super happy to have same Pages & Components structure in two places so any suggestions would be great!

You mean packages/grafana-e2e/src/components/index.ts ?

**Comment by stevenvachon** [comment]
Hmm, so @grafana/e2e is now only a CLI that wraps cypress?

**Comment by hugohaggmark** [comment]
> > Not super happy to have same Pages & Components structure in two places so any suggestions would be great!
> 
> You mean packages/grafana-e2e/src/components/index.ts ?

I made it so the e2e-selectors package are master for all e2e objects. See my latest commits.

**Comment by hugohaggmark** [comment]
> Hmm, so @grafana/e2e is now only a CLI that wraps cypress?

I don't know; I didn't add the CLI part so I know little about that but the main reasons because we have grafana/e2e is to make it very easy to create e2e tests for Grafana and the only constraint we had was to keep the framework very thin and small. In that perspective I would say it does a good job but if you think differently, please leave some feedback.

File: packages/grafana-e2e-selectors/src/selectors/index.ts [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import { Pages } from './pages';
+import { Components } from './components';
+
+export const selectors = {
```
[the remark this answers is not in the document]
Comment (hugohaggmark) [reply packages/grafana-e2e-selectors/src/selectors/index.ts]: Nope :)

[the remark this answers is not in the document]
Comment (dprokop) [reply packages/grafana-e2e-selectors/src/selectors/index.ts]: haha, OK :D

**Review by dprokop — APPROVED** [review]
🚀

**Pull request merged**